demo/gatsby-config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
           appId: process.env.FIREBASE_APP_ID,
         },
         loginPath: "/login",
-        socialLogins: ["google", "github"],
+        socialLogins: ["google", "github", "facebook"],
```

Comment:
Let's not add new socialLogins to the demo. You can do this locally, but please don't commit it.